frontend/src/components/GameGallery.js

```diff
@@ -1,100 +1,220 @@
 import React, { useState } from 'react';
 import GameThumbnail from "./GameThumbnail";
+import { BsArrowLeft } from "react-icons/bs";
 import '../styles/GameGallery.css';
+import games from '../games.json'
```

Comment:
@m-dorsey could you change this to use the games.json file in the 'data' directory? It will be empty because it's meant to store the pi's data, but you can put mock data in it for your testing

Reply:
this has been completed :)